platform/ios/src/MGLMapViewDelegate.h

```diff
@@ -234,17 +234,20 @@ NS_ASSUME_NONNULL_BEGIN
  A value of `0.0` results in a completely transparent shape. A value of `1.0`,
  the default, results in a completely opaque shape.
  
+ @deprecated Use `-mapView:strokeColorForShapeAnnotation:`.
```

Comment:
You’re right, jazzy doesn’t support `@deprecated` — I removed the doc entirely in 2b1bde35f3c60dd2f68130ce2d9b4c224aee6b8c.